infra/scripts/container-scripts/destroy-dbs.sh

```
@@ -8,14 +8,16 @@ export PGPASSWORD
 set -x
 FEATURE_NAME=$1
 
-psql -tc "SELECT datname FROM pg_database WHERE datname like 'feature_${FEATURE_NAME}_%'" --field-separator ' ' --no-align --quiet |
-  while read -r dbname; do
-    psql -c "DROP DATABASE $dbname"
-  done
+echo "feature name is $feature"
```

Shellcheck (0.10.0)

[warning] feature is referenced but not assigned.

(SC2154)